### DISCUSSION TITLE
Phyton error when trying to install CP

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#93] [General]
I have the following error when trying to install CP on a Centos 7.9 release...  Seems is related to phyton,  not sure if I have to install phyton (and what release 2, or 3) first, and where  (not sure if on the ansible machine or in the CP machinetall)... Should i have phyton for CP install ?  

$ ansible-playbook -i environments/examples/cnx7/quick_start/component_pack playbooks/setup-component-pack-complete-single-node.yml

PLAY [Setup Nginx] *************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************
ok: [web.giahsa.com]

TASK [nginx-install : Prepare OS for Nginx] ************************************************************************************
included: /home/ansible/connections-automation/roles/third_party/nginx-install/tasks/setup_os.yml for web.giahsa.com

TASK [nginx-install : Install pre-requisites Yum for CentOS 7 - epel-release] **************************************************
skipping: [web.giahsa.com]

TASK [nginx-install : Apply repository for RHEL 7] *****************************************************************************
ok: [web.giahsa.com]

TASK [nginx-install : Install prerequisits for semanage] ***********************************************************************
failed: [web.giahsa.com] (item=libselinux-python) => {"ansible_loop_var": "item", "changed": false, "failures": ["libselinux-python All matches were filtered out by modular filtering for argument: libselinux-python"], "item": "libselinux-python", "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
failed: [web.giahsa.com] (item=libsemanage-python) => {"ansible_loop_var": "item", "changed": false, "failures": ["No package libsemanage-python available."], "item": "libsemanage-python", "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}

PLAY RECAP *********************************************************************************************************************
web.giahsa.com             : ok=3    changed=0    unreachable=0    failed=1    skipped=1    rescued=0    ignored=0

**Comment by dejo1307** [comment]
This sounds like regression introduced by https://github.com/HCL-TECH-SOFTWARE/connections-automation/pull/74, let me retest it and revert it if the fix is needed.

**Reply by dejo1307** [reply]
I couldn't reproduce it, worked fine. 

Can you run yum update on that machine (web.giahsa.com) and try it again?

**Reply by agorosti** [reply]
Thks dejan.

It seems it was relationated with the OS release  (a RedHat 8.3).  a yum update did not worked either,  but I deleted the AMI Image on AWS,  and created a new one based on Centos 7.9,   configured ansible again on that new instance,  and I was able to launch the playbook and progress ahead ..  If I got new errors on the following taks,  I will open a new topic.
